Small fixes (various)

MeuhV1 opened the pull request:
Fixed missing TXT key for Moors DCN
Fixed TXT key for Polynesian DCN
Fixed error in RFCUtils at makeUnit
Fixed bug when renaming city when being Harappa
Fixed Colombian leader
Fixed Civic screen resolution for lower screen resolutions (It's slighty smaller as well on higher resolutions, but it's still very well readable)
Pillaging with Corsair counts towards UHV

dguenms committed:
Fix CNM

Commit:
Fixed TXT key for Moors
Fixed error in RFCUtils at makeUnit
Fixed bug when renaming city when being Harappa
Fixed Colombian leader

Commit:
Fixed TXT key for Moors
Fixed error in RFCUtils at makeUnit
Fixed bug when renaming city when being Harappa
Fixed Colombian leader

Commit:
Small potential (very unlikely) CTD fix

Commit:
Small potential (very unlikely) CTD fix

SgtBears committed:
Fixes to Dynamic Names

SgtBears committed:
A few I missed

Commit:
Fixed civic screen for lower resolutions
(It's a bit smaller on higher resolutions, but it's still very well readable)

Commit:
Fixed civic screen for lower resolutions
(It's a bit smaller on higher resolutions, but it's still very well readable)

Commit:
Polynesian DCN fixes

Commit:
Pillaging with Corsairs do count towards Moorish UHV

Commit:
Pillaging with Corsairs do count towards Moorish UHV

dguenms committed:
Merge pull request #58 from dguenms/python-constants

Python constants

dguenms committed:
Merge pull request #57 from SgtBears/Dynamic_Name_Fixes

Dynamic Name Fixes

dguenms commented on Assets/Python/CvRFCEventHandler.py:
```diff
@@ -453,8 +453,9 @@ def onCityGrowth(self, argsList):
-		if iPlayer == iVikings and iGold > 0 and iImprovement != -1 and iGold < 1000:
-			vic.onUnitPillage(iPlayer, iGold)
+		iUnit = unit.getUnitType()
+		if (iPlayer == iVikings or iPlayer == iMoors) and iGold > 0 and iImprovement != -1 and iGold < 1000:
```
The Moors do not have a pillage related goal, what is the idea behind this?

dguenms commented on Assets/Python/DynamicCivs.py:
```diff
@@ -1784,6 +1784,10 @@ def leader(iPlayer):
 	elif iPlayer == iRome:
 		if not bEmpire and not bCityStates: return iAugustus
 		
+	elif iPlayer == iMaya:
```
Good catch.

dguenms commented on Assets/Python/RFCUtils.py:
```diff
@@ -282,7 +282,7 @@ def makeUnit(self, iUnit, iPlayer, tCoords, iNum, sAdj="", iExp = 0): #by LOQ
 				unit.changeExperience(iExp, 100, False, False, False)
 				
-		return unit
+		#return unit
```
Yes, it's probably better to remove that.

dguenms commented on Assets/Python/Victory.py:
```diff
@@ -1848,12 +1848,16 @@ def onGreatPersonBorn(iPlayer, unit):
 		if isPossible(iVikings, 2):
 			sd.changeVikingGold(iGold)
+			
+	elif iPlayer == iMoors:
```
Oh, I see. The Moorish goal is intended to only count sunk ships, is the goal phrased differently?

dguenms commented on Assets/XML/Text/DynamicNames/DynamicNames_Moors.xml:
```diff
@@ -40,4 +40,8 @@
 		<English>Hejaz</English>
 	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_CIV_MOORS_EMIRATE_OF</Tag>
```
Should be Emirate of %s1.

dguenms commented on Assets/XML/Units/CIV4PromotionInfos.xml:
```diff
@@ -4184,7 +4184,7 @@
 			<bShiftDown>0</bShiftDown>
 			<bCtrlDown>0</bCtrlDown>
 			<iHotKeyPriority>0</iHotKeyPriority>
-			<Button>,Art/Interface/Buttons/Promotions/Logistics1.dds</Button>
```
Are you sure this is necessary? The buttons do work in the game don't they? I think I know of crashes/loading problems without the leading comma in some systems. (Same for below.)

dguenms commented:
The branch is out of synch (which is probably my fault for delaying the review for so long), can you do a rebase after addressing the other comments so I can merge this?

Commit:
Rebased
Removed Emirate TXT_KEY (sgt Bears already fixed this)

Commit:
Merge branch 'SmallFixes' of https://github.com/MeuhV1/Dawn-of-Civilization into SmallFixes

Commit:
Rebased to consts to python updates
Removed TXT key for Moorish Emirate (sgt bears fixed it already)

MeuhV1 commented:
I rebased the branch.

The TXT key for the Moors were fixed by sgt bears in the meantime, so I removed it.

IIRC, the comma in fromt on the buttons cause a CTD if that button appears in the citybar. As you can't build promotions, this will not appear. So it isn't really necessary. But this avoids any risk. (Maybe the comma can cause a CTD for the same reason in another situation we didn't encounter yet)

I just think that pillaging  sea improvements is also a part of pirating. But if you don't think it is suitable, feel free to ignore it. (It was more of a suggestion than a real fix)

dguenms commented:
Okay, I looked into the comma stuff again and you're right. This is the safer way of doing it.

I'll think about the Moorish UHV goal. In the very least, the description should probably be more specific.
